test/core/run_tests.py

```diff
@@ -95,6 +95,7 @@ def run_cmd(mode):
                     'LC_ALL': 'C.UTF-8',
                     'PYTHONIOENCODING': 'utf_8',
                     'PATH': os.environ.get('PATH', '.'),
+                    'PYTHONIOENCODING': 'utf_8',
```

Comment:
`PYTHONIOENCODING` seems to be defined twice

Reply:
will fix